src/scripts/updateData_REST.sjs

```diff
@@ -92,7 +96,8 @@ try {
         if (!(EA.version in oldDoc.root.fullDetails.ea_version)) {
           json.fullDetails.ea_version.push(oldDoc.root.fullDetails.ea_version[0]);
         }
-
+        
+        xdmp.log('replacing document with: ' + JSON.stringify(json));
```

Comment:
please remove this line -- it's a lot of information to dump in the log after the debugging is done